java/src/main/kotlin/io/spine/validation/java/setonce/SetOnceMessageField.kt

```diff
@@ -50,9 +51,9 @@ internal class SetOnceMessageField(
         }
     }
 
-    private val fieldTypeClass = field.type.message
-        .javaClassName(declaredIn.fileHeader)
-        .canonical
+    private val fieldTypeClass =
+        field.type.message.toMessageType(typeSystem).javaClassName(typeSystem)
+            .canonical
```

Comment:
```suggestion
    private val fieldTypeClass = field.type.message
            .toMessageType(typeSystem)
            .javaClassName(typeSystem)
            .canonical
```

The same suggestion.